src/main/java/net/dries007/tfc/world/surface/builder/DuneSurfaceBuilder.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Licensed under the EUPL, Version 1.2.
+ * You may obtain a copy of the Licence at:
+ * https://joinup.ec.europa.eu/collection/eupl/eupl-text-eupl-12
+ */
+
+package net.dries007.tfc.world.surface.builder;
+
+
+import net.dries007.tfc.world.surface.SoilSurfaceState;
+import net.dries007.tfc.world.surface.SurfaceBuilderContext;
+import net.dries007.tfc.world.surface.SurfaceState;
+
+public class DuneSurfaceBuilder implements SurfaceBuilder
+{
+    public static final SurfaceBuilderFactory INSTANCE = DuneSurfaceBuilder::new;
+
+    protected DuneSurfaceBuilder(long seed) {}
+
+    @Override
+    public void buildSurface(SurfaceBuilderContext context, int startY, int endY)
+    {
+        context.setSlope(0);
```

Comment:
Why? If the biome is exceedingly flat, this shouldn't be necessary, and this seems like it would cut off borders where the slope is intended to interpolate between them. imo it would be better practice to check the slope, and use a different surface builder vs. setting the slope whenever it's used.